code/datums/status_effects/debuffs/spacer.dm

```diff
@@ -40,9 +40,9 @@
 
 	if(seconds_in_nograv >= 3 MINUTES)
 		// This has some interesting side effects with gravitum or similar negating effects that may be worth nothing
-		owner.adjustStaminaLoss(-1 * stamina_heal_per_tick)
+		owner.adjustStaminaLoss(-3 * stamina_heal_per_tick)
```

Comment:
1. Why are you changing the the multiplier and not the var
2. I don't think spacers should be getting passive 9 stamina regen per tick, that seems too much

Reply:
1. Because I can't read good and I wanna learn to do other stuff good
2. Spacer isn't good enough to cost 7 trait points but, I agree. This was meant to make it 2 higher per tick, not 9...

Reply:
actually just leaving it how it was; 3 stam per tick seems reasonable for something that you could have perpetually if you blow up the grav gen or do the unthinkable of actually getting wittel